resources/l10n/zh_CN/LC_MESSAGES/loot.po

```diff
@@ -68,7 +68,7 @@ msgstr "排序加载顺序..."
 
 #: src/gui/state/loot_state.cpp:105
 msgid "Error: --auto-sort was passed but no --game parameter was provided."
-msgstr "错误: --自动排序已经完成但是没有 --提供游戏参数。"
+msgstr "Error: --auto-sort was passed but no --game parameter was provided."
```

Comment:
@silentdark Only `--auto-sort` & `--game` need to stay untranslated. The rest of the sentence can be translated.